client/css/style.css

```diff
@@ -678,9 +678,15 @@ button, .user {
 }
 #chat .part .from:before,
 #chat .quit .from:before {
-    font-family: FontAwesome;
+	font-family: FontAwesome;
 	content: "\f08b";
 	color: #FF4136;
+	display: inline-block;
+	-webkit-transform: rotate(180deg);
+	-moz-transform: rotate(180deg);
+	-ms-transform: rotate(180deg);
+	-o-transform: rotate(180deg);
+	transform: rotate(180deg);
```

Comment:
Oh, I'm not suggesting we should do it just now, I'm just saying that if we were to do it (which is a whole different discussion) then we would use auto-prefixer, not do it manually.